measures/HEScoreHPXML/measure.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# see the URL below for information on how to write OpenStudio measures
+# http://nrel.github.io/OpenStudio-user-documentation/measures/measure_writing_guide/
+
+require 'openstudio'
+require 'open3'
+
+# start the measure
+class HEScoreHPXML < OpenStudio::Measure::ModelMeasure
```

Comment:
Seems like this measure should have some unit tests.

Reply:
Decided not to add unit tests due to the complexity involved in calling the external translator